android/src/main/java/com/reactNativeQuickActions/AppShortcutsModule.kt

```diff
@@ -0,0 +1,170 @@
+package com.reactNativeQuickActions
+
+import android.annotation.TargetApi
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.ShortcutManager
+import android.net.Uri
+import android.os.Build
+import android.util.Log
+import androidx.core.content.IntentCompat
+import androidx.core.content.pm.ShortcutInfoCompat
+import androidx.core.content.pm.ShortcutManagerCompat
+import androidx.core.graphics.drawable.IconCompat
+import androidx.core.net.toUri
+import com.facebook.react.bridge.ActivityEventListener
+import com.facebook.react.bridge.Callback
+import com.facebook.react.bridge.JSApplicationIllegalArgumentException
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableArray
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.modules.core.DeviceEventManagerModule
+import com.reactNativeQuickActions.ShortcutItem.Companion.fromReadableArray
+
+@ReactModule(name = AppShortcutsModule.REACT_NAME)
+internal class AppShortcutsModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+    init {
+        reactContext.addActivityEventListener(object : ActivityEventListener {
+            override fun onActivityResult(activity: Activity, requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
Can't you just omit this code if you don't do anything?

Reply:
I did not change this code (it just shows up as new because I migrated the file from Java to Kotlin)

But also I could not remove this, as it would result in an error:

![image](https://github.com/Loxone/react-native-quick-actions/assets/7384552/deb7211d-424e-4c85-83b6-49f6dd10f548)